packages/atomic/src/components/common/item-list/styles/mixins.pcss

```diff
@@ -215,6 +215,9 @@
           grid-template-columns: repeat(3, 1fr);
         }
         @media (min-width: 1024px) {
+          grid-template-columns: repeat(3, 1fr);
```

Comment:
This new breakpoint prevents the variant icons from being too small, and other issues due to lack of space.